.travis.yml

```diff
@@ -18,9 +18,7 @@ services:
 before_install:
   - git clone https://github.com/oceanprotocol/barge
   - cd barge
-  - export PROVIDER_VERSION=ptalgs
   - if [[ -n "$DOCKERHUB_PASSWORD" && -n "$DOCKERHUB_USERNAME" ]]; then echo "logging in to docker";echo "$DOCKERHUB_PASSWORD" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin;fi
-  - export AQUARIUS_VERSION=v2.2.5
```

Comment:
This change doesn't seem to "strip JSON spaces", and there is no context to what it's for. 

Could you either update the PR description (and ensuing commit message) to give context, or put this in a different PR?